packages/tldraw/src/index.ts

```diff
@@ -170,6 +133,96 @@ export {
 	type TldrawFile,
 } from './lib/utils/tldr/file'
 
+// Minimap default component
+export { DefaultMinimap } from './lib/ui/components/Minimap/DefaultMinimap'
+
+// General UI components for building menus
+export { TldrawUiMenuCheckboxItem } from './lib/ui/components/menus/TldrawUiMenuCheckboxItem'
+export { TldrawUiMenuContextProvider } from './lib/ui/components/menus/TldrawUiMenuContext'
+export { TldrawUiMenuGroup } from './lib/ui/components/menus/TldrawUiMenuGroup'
+export { TldrawUiMenuItem } from './lib/ui/components/menus/TldrawUiMenuItem'
+export { TldrawUiMenuSubmenu } from './lib/ui/components/menus/TldrawUiMenuSubmenu'
+export { unwrapLabel } from './lib/ui/hooks/useActions'
+
+export {
+	TldrawUiComponentsProvider,
+	useTldrawUiComponents,
+	type TLUiComponents,
+} from './lib/ui/hooks/useTldrawUiComponents'
+
+export { DefaultPageMenu } from './lib/ui/components/PageMenu/DefaultPageMenu'
+
+export {
+	DefaultStylePanel,
+	type TLUiStylePanelProps,
+} from './lib/ui/components/StylePanel/DefaultStylePanel'
+export {
+	DefaultStylePanelContent,
+	type TLUiStylePanelContentProps,
+} from './lib/ui/components/StylePanel/DefaultStylePanelContent'
+
+// Toolbar
+export { DefaultToolbar } from './lib/ui/components/Toolbar/DefaultToolbar'
+
+// Actions menu
+export {
+	DefaultActionsMenu,
+	type TLUiActionsMenuProps,
+} from './lib/ui/components/menus/ActionsMenu/DefaultActionsMenu'
```

Comment:
feels like there shoud be a `/lib/ui/components/index.tsx` or `/lib/ui/components/menus/index.tsx` export list and this should all re-export from that. thoughts?

Reply:
generally we avoid barrel files, some bundlers have tripped over them in the past

Reply:
fair point — yeah that's true.